CONTRIBUTING.md

```diff
@@ -128,9 +118,7 @@ This process is a little ridiculous...
 
 -   Build and upload the new version
 
-    ```shell
-    ./gradlew clean :uploadArchives
-    ```
+-   Update the version number, tag a release, and upload an archive by running `make VERSION=[number] release`
 
 -   "Promote" the release build on Maven Central
```

Comment:
If you want to go further, I had some luck with the [gradle nexus staging plugin](https://github.com/Codearte/gradle-nexus-staging-plugin) - it replaces all of the maven central manual steps. [bugsnag-java release instructions](https://github.com/bugsnag/bugsnag-java/blob/master/CONTRIBUTING.md#3-making-a-release)